pkg/controllers/sync/history.go

```diff
@@ -158,7 +158,8 @@ func (s *SyncController) updateRevisionLabel(
 		clone.Revision = 0
 		if _, err := s.controllerHistory.UpdateControllerRevision(clone, revisionNumber); err != nil {
 			if apierrors.IsNotFound(err) {
-				klog.V(4).Infof("failed to update the revision %s with error %v", revision.Name, err)
+				keyedLogger.WithValues("controllerRevision-name", revision.Name).
```

Comment:
Kebab case MUST be used for keys of structured logging fields.

Reply:
done, pls take review.